### ISSUE TITLE
Date Picker: Accessibility for Month & Year

### ISSUE THREAD

**Issue by sparkypro** [SAP/ui5-webcomponents#5422]
**Describe the bug**
The ‘Next’ and ‘Previous’ buttons within the calendar are not reachable via keyboard

**UI Link**
visit https://sap.github.io/ui5-webcomponents-react/?path=/docs/inputs-datepicker--default-story

**To Reproduce**
1. Go to DatePicker
2. Use keyboard tab key to reach month
3. The table of months is not read out with screen reader
4. The table of year is not read out with screen reader

**Expected behavior**
The table of month or year must be read out by screen reader

**Screenshots**
<img width="103" alt="image" src="https://user-images.githubusercontent.com/108118034/176108592-d2335e61-a1a5-4b7c-97fd-2b6886e9134b.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version:1.3.1
`@ui5/webcomponents-react` version: 0.23.2
Operating System: Windows 11
Browser: Chrome

**Comment by nnaydenow** [comment]
Hello @SAP/ui5-webcomponents-topic-b,

Could you please check this issue? In OpenUI5 when the focus is inside month / year picker area the name of area is announced (https://openui5.hana.ondemand.com/entity/sap.m.DatePicker/sample/sap.m.sample.DatePicker) but ui5-date-picker it is not (https://sap.github.io/ui5-webcomponents/playground/components/DatePicker/)

Regards,
Nayden

**Comment by unazko** [comment]
Hello @sparkypro,

The "Next" and "Previous" buttons are not reachable via keyboard by design.
The screen reader users have easier ways to navigate back and forth by using the following keyboard shortcuts:
- Page Up/Down (Navigates with one month, one year or one range of years depending on the current picker)
- Shift + PageUp/Down (When in days view navigates back and forth by one year) 
- Ctrl + Shift + PageUp/Down (When in days view navigates back and forth by ten years)

Therefore I'm closing the incident.

Best regards,
Boyan
